### PR TITLE
Adds defibs to Sci

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Adds compact and normal defibs into sci nodes
They are costly and spawn without a cell, you need both gold and silver to make them as well as basic metal/glass/plastic 
Revival is kinda wanted in game for when ~~Hank~~ people walk into emitter shots and stand, or when someone deds from spider/roaches in maints

### CODE REVIEW

File: code/modules/research/designs.dm [hunk 1]
```diff
@@ -95,6 +90,18 @@ other types of metals and chemistry for reagents).
...
 	sort_string = "MACBB"
 
+/datum/design/research/item/medical/defibs
+	desc = "Large paddles that get charged via a cell to give a controlled shock for revival."
+	req_tech = list(TECH_BIO = 4, TECH_POWER = 2)
+	build_path = /obj/item/device/defib_kit
+	sort_string = "MACBC"
+
+/datum/design/research/item/medical/compact_defibs
+	desc = "Much smaller defibulators build into a belt, to more easly carry."
```
Comment (DogZeroX) [inline code/modules/research/designs.dm]: You misspelled _Easily_.

File: code/game/objects/items/devices/defib.dm [hunk 2]
```diff
@@ -177,6 +178,7 @@
 	item_state = "defibcompact"
 	w_class = ITEM_SIZE_NORMAL
 	slot_flags = SLOT_BELT
+	matter = list(MATERIAL_STEEL = 5, MATERIAL_PLASTIC = 7, MATERIAL_GLASS = 2, MATERIAL_GOLD = 7.75, MATERIAL_SILVER = 12.75) //Glass for the lights
```
Comment (DogZeroX) [inline code/game/objects/items/devices/defib.dm]: Why does it take more gold and silver if it's smaller?

Comment (Trilbyspaceclone) [reply code/game/objects/items/devices/defib.dm]: better wiring is the idea. Thats why it takes mag 7 and not EMP tech

Comment (king5327) [reply code/game/objects/items/devices/defib.dm]: The compact defibrillator is worse than the regular defibrillator, but in exchange it's portable. It only gets like 4 shocks on a battery, does a very tiny amount of oxyloss healing, but uses a medium battery and less charge, compared to the large battery (and higher charge cost) of the standard size.

Comment (Trilbyspaceclone) [reply code/game/objects/items/devices/defib.dm]: We still get tons of the mats their, its nothing like iron, or what ever the durasteel is. Its still far and costly as intended for balance and use for miners

Comment (king5327) [reply code/game/objects/items/devices/defib.dm]: I still stand by my position that the compact defib is weaker. It should need less plastic and steel, maybe more gold and silver for efficiency and whatnot.

Comment (Trilbyspaceclone) [reply code/game/objects/items/devices/defib.dm]: It... Is that...

**Pull request merged**